common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -59,12 +60,14 @@ private LeaderInquireClient(String zookeeperAddress, String leaderPath) {
         CuratorFrameworkFactory.newClient(mZookeeperAddress, new ExponentialBackoffRetry(
             Constants.SECOND_MS, 3));
     mCLient.start();
+
+    mMaxTry = new TachyonConf().getInt(Constants.ZOOKEEPER_LEADER_INQUIRY_RETRY_COUNT);
```

Comment:
good catch. yes this should be `ClientContext.getConf()` rather than `new TachyonConf`. Otherwise, there will be problems potentially.